src/lib/css-parse.html

```diff
@@ -89,6 +90,12 @@
       return node;
     },
 
+    // conversion of sort unicode escapes with spaces `\33 ` into
+    // expanded form that doesn't require space `\000033`
+    _expandUnicodeEscapes : function (s) {
+      return s.replace(/\\([0-9]+)\s/g, '\\0000$1');

```

Comment:
Thid is not correct for the general case. 

-  Should be `[0-9a-fA-F]` -- escapes are hexadecimal.
-  Hex number needs to be limited to six hex characters, otherwise you could match a valid, non-space-delimited escape, followed by letters and a meaningful space, like `\0000E9da `.
-  Last and most challenging issue: you can't assume the unicode escape is exactly two digits. 3,4,5 and 6 digit escapes followed by a space are valid (for 6-digit escapes, the space is optional, but we'd need to consume it here).

More information here:

http://www.w3.org/International/questions/qa-escapes#cssescapes